### PR TITLE
Main view has only 3 buttons for reporting

### LINKED ISSUE #100: Main view should only be 3 buttons for reporting
Change Heading to a single title (translate): `What would you like to report?`

3 rounded colored buttons:
- Adult: color `DFD458` 40%
- Bite: color `D28A73` 40%
- Breeding site: color `7D9393` 40%

Move "My data" to a the navigationdrawer

Screenshot from 2024-03-25 09-43-43

### PR DESCRIPTION
**Pull request by DigitalSeneca**
fixes #100

- I would need to receive the translation string for "What's in your surroundings?" or "What would you like to report?"

image

### CODE REVIEW

**Review by epou — APPROVED** [review]
Brilliant! I'll update the 'what_to_do_txt' localized string once the team decides on the final question.

**Comment by epou** [comment]
Last news! Team has voted for a randomized text choice (from a list of 5) on every home_page load.

Could you prepare the code to get a random integer between 1 and 5 on every home_page load/init and get it's corresponding localized text from: `what_to_do_txt_{random_number_choice}`
Options will be:
- what_to_do_txt_1
- what_to_do_txt_2
- what_to_do_txt_3
- what_to_do_txt_4
- what_to_do_txt_5

File: lib/pages/main/home_page.dart [hunk 1]
```diff
@@ -36,7 +32,7 @@ class _HomePageState extends State<HomePage> {
   void initState() {
     super.initState();
     initAuthStatus();
-    //_getNotificationCount();
+    _getNotificationCount();
```
Comment (epou) [inline lib/pages/main/home_page.dart:35]: I guess this method is already in main_vc.dart from pull https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/pull/141

File: lib/pages/main/home_page.dart
Comment (epou) [inline lib/pages/main/home_page.dart:1]: Remove the following:
```dart
 String? userName;
  StreamController<String?> nameStream = StreamController<String?>.broadcast();
  String? userUuid;
int unreadNotifications = 0;
void _getData()
void _getNotificationCount()
void updateNotificationCount
```

File: assets/language/mk_MK.json
Comment (epou) [inline assets/language/mk_MK.json:1]: Remove this

Comment (DigitalSeneca) [reply assets/language/mk_MK.json:1]: that mk_MK thing is annoying because I can't automatically download them from localise 😅 I'm removing it now

Comment (epou) [reply assets/language/mk_MK.json:1]: Let me discuss about removing mk_MK at tomorrow's meeting

**Pull request merged**